### ISSUE TITLE
Are `urn` and `label` properties or something else?

### ISSUE THREAD

**Issue by Eumaeus** [cite-architecture/citeobj#40]
An object's `urn` and `label` are recorded as properties in the `.cex` file, but do not get included in the Vector returned by PropertyList. 

This may be just fine… but it confused me for a while. It might be worth thinking about whether we want to include those with all the other properties in the data model.

**Comment by neelsmith** [comment]
No, we don't.  The abstract model of a citable object is precisely:

1. it is identified by a URN
2. it has a human-readable label
3. it has 0 or more typed properties

The scala implementation expresses this very accurately.  The fact that it's convenient to serialize this in a single delimited-text record per object is not a criticism of the model or the scala implementation of the model.  (It may be a valid criticism of the plain-text representation.  I'm inclined to favor a simple row-per-object model, even though it means URNs and labels are not visually distinguished from collection-specific properties, but if you think that's a problem, let's pick that topic up.

Leaving this issue open as an issue for *documentation* --  need to explain the model of citable obect somewhere -- but *not* as an issue for changing the model or its scala representation.
